maps/away/errant_pisces/errant_pisces_areas.dm

```diff
@@ -86,7 +86,7 @@
 	icon_state = "fishing_wing"
 
 /area/errant_pisces/bridge
-	name = "Bridge"
+	name = "Errant Pisces Bridge"
```

Comment:
Why not use the sector name logic for ships as well, so that the name of the ship is added instead of `Errant Pisces`? Because that's not a player-facing name, `CV Ahab's Harpoon` is.

Reply:
It's used only for sectors. Compare `/obj/effect/overmap/visitable/sector` and `/obj/effect/overmap/visitable/ship`. `overmap_sectors` contains any subtype of `/obj/effect/overmap/visitable`, I believe.

Reply:
Nevermind, my bad.